js/chat_app/src/services/utils/utils.js

```diff
@@ -0,0 +1,27 @@
+export const query = async (url, method, authToken, body = {}) => {
+  try {
+    if (Object.keys(body).length) {
+      const data = await fetch(url, {
+        method,
+        headers: {
```

Comment:
you can make headers as separate object

like
`
const headers = {
          'content-type': 'application/json',
          'Auth-Token': authToken
}
`